### DISCUSSION TITLE
Providing one-off support without configuring network settings

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk-server-pro#221] [Q&A]
Unless I'm missing something obvious there appears to be no way of providing one-off support without having the end user configure the network ID / relay server settings. 

Is there a way around this?

**Comment by ghost** [comment]
Anyone?

**Comment by AlvaroNieto** [comment]
Create a custom client if you have updated the server to 1.2.0. Check the videos un the release page:

https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.2.0

Then place the file in someplace reacheable to the user. Make them download the new support executable, it will have the settings ready.
